src/jit/emit.h

```diff
@@ -2174,6 +2174,11 @@ class emitter
                               WORD  slotNum   = 0,  /* IN */
                               INT32 addlDelta = 0); /* IN */
 
+#ifdef _TARGET_ARM_
+    void emitHandlePCRelativeMov32(void* location, /* IN */
+                                   void* target);  /* IN */
+#endif
```

Comment:
Format checker required this line to be the following (maybe, because of comments on the previous lines):
```
#endif                                             // _TARGET_ARM_
```

Please, suggest, how is it better to workaround the issue in format checker.